package.json

```diff
@@ -65,6 +65,8 @@
         "@types/node": "11.13.0",
         "canvas": "2.5.0",
         "svgdom": "^0.1.8",
-        "webpack-node-externals": "^2.5.2"
+        "webpack-node-externals": "^2.5.2",
+        "node-fetch": "^2.6.1",
```

Comment:
@hritikaggarwal31 Are you using these dependencies? I think these break in the dashboard. Please remove them if not required.